### DISCUSSION TITLE
Tool to batch export timeline clips with handles?

### DISCUSSION THREAD

**Discussion by tangierc** [CommandPost/FCPCafe#177] [Final Cut Pro Questions]
Anyone know of a tool that'll let me batch export clips on the timeline with handles from Final Cut Pro?

**Comment by latenitefilms** [comment]
The best/easiest/cheapest option is just to export a FCPXML and send it to DaVinci Resolve.

**Comment by benblaser60** [comment]
I’m not sure but Lumberjack may have a tool for that.
